### DISCUSSION TITLE
Is a way to SEND RAW ETHERNET PACKET?

### DISCUSSION THREAD

**Discussion by matiaskenig** [Networking-for-Arduino/EthernetENC#36] [General]
Hi Friend JAndrassy, nice work with the lib, its amazin.
I have a question for you: its a way to send RAW ethernet Packet?

**Comment by JAndrassy** [comment]
what is a RAW Ethernet packet?

**Comment by matiaskenig** [comment]
Hi!, thank you for reply!!!, im impressed with your work.
RAW packet is a packet without specific tcp or udp header (maybe it has
another name, I know it that way)
Some specific hardware use this kind of "hardcoded" kind of data (It is a
way to keep your customers captive)
i have some data captured in wireshark of some special screens, if you want
i can send it to you
in my experience, the only way to send this type of frame was on a pc with
a winpcap library
It would be really useful to be able to send this type of frames with your
library, since it would open the doors to everyone to be able to interact
with these devices

El mié, 25 may 2022 a la(s) 01:43, Juraj Andrássy ***@***.***)
escribió:

> what is a RAW Ethernet packet?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JAndrassy/EthernetENC/discussions/36#discussioncomment-2816916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF276MKO2BZ4QDMVFY6R63DVLWVVFANCNFSM5W3KDQIA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Matías A. Kenig - Dpto. Sistemas
Cel: 15-2607-9550 / Nextel: 546*813

**Comment by JAndrassy** [comment]
The library is not my work. I only fixed a ton of bugs and I maintain it. 
Enc28J60Network.cpp communicates with the enc28j60 over SPI. The protocol is described in the enc28j60 datasheet

**Comment by matiaskenig** [comment]
Thank you Andrassy

El mié, 25 may 2022 a la(s) 13:54, Juraj Andrássy ***@***.***)
escribió:

> The library is not my work. I only fixed a ton of bugs and I maintain it.
> Enc28J60Network.cpp communicates with the enc28j60 over SPI. The protocol
> is described in the enc28j60 datasheet
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JAndrassy/EthernetENC/discussions/36#discussioncomment-2821804>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF276MMZ3DQLS6E4DMU6ULDVLZLLVANCNFSM5W3KDQIA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Matías A. Kenig - Dpto. Sistemas
Cel: 15-2607-9550 / Nextel: 546*813
